extensions/BugModal/web/new_bug.js

```diff
@@ -1,28 +1,28 @@
+var initial = {}
 var comp_desc = {}
 
-function load_products(query, callback) {
+$(document).ready(function() {
     bugzilla_ajax(
             {
-                url: 'rest/bug_modal/products'
+                url: 'rest/bug_modal/initial_loader'
```

Comment:
'init' would also work well here, though it may be too abrupt/brief for practical use.